### PR TITLE
109 display live data (Presentation)

### LINKED ISSUE #109: display live data (Presentation)

### PR DESCRIPTION
**Pull request by Kaszz**
Add live view for all relevant variables of the machine

close #109

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #Datasource setup
 spring.datasource.url=jdbc:postgresql://localhost:5432/brewmes
 spring.datasource.username=postgres
-spring.datasource.password=#password
+spring.datasource.password=0205
```
Comment (Wylardt) [inline src/main/resources/application.properties:4]: Laust will fucking murder you

File: WebApplication/package.json [hunk 2]
```diff
@@ -40,5 +40,17 @@
...
+  "devDependencies": {
+    "@iconify/icons-bx": "^1.1.0",
+    "@iconify/icons-cil": "^1.1.0",
+    "@iconify/icons-emojione": "^1.1.0",
+    "@iconify/icons-emojione-v1": "^1.1.0",
+    "@iconify/icons-jam": "^1.1.0",
+    "@iconify/icons-mdi": "^1.1.0",
+    "@iconify/icons-ri": "^1.1.0",
+    "@iconify/icons-subway": "^1.1.0",
+    "@iconify/react": "^1.1.3",
```
Comment (Lawst96) [inline WebApplication/package.json:53]:
```suggestion
    "@iconify/icons-bx": "^1.1.0",
    "@iconify/icons-cil": "^1.1.0",
    "@iconify/icons-emojione": "^1.1.0",
    "@iconify/icons-emojione-v1": "^1.1.0",
    "@iconify/icons-jam": "^1.1.0",
    "@iconify/icons-mdi": "^1.1.0",
    "@iconify/icons-ri": "^1.1.0",
    "@iconify/icons-subway": "^1.1.0",
    "@iconify/react": "^1.1.3",
```
Remove unused dependencies

**Review by Lise4070 — APPROVED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (Wylardt) [reply src/main/resources/application.properties:4]: I don't care tho

**Review by Wylardt — APPROVED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (Lawst96) [inline src/main/resources/application.properties:4]:
```suggestion
```
🙄🙄🙄🙄

**Review by Lawst96 — CHANGES_REQUESTED** [review]

**Review by Lawst96 — APPROVED** [review]

**Pull request merged**